server/manageuser.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+require_once 'config.php';
+require_once "user.class.php";
+
+$USER = new User();
+if (!$USER->hasPermission('edit_user_permission')) {
+	header("Location: /unauthorized");
+	die();
+}
+
+if(isset($_REQUEST['callback'])){
```

Comment:
On a sidenote, is this the preferred way of handling multiple request types through the same file? This 'callback' variable along with the request?

Reply:
Not sure on that, I've done it before as needed but it's usually taken care of by a framework in my experience.